### PR TITLE
[Issue #2496] Run vuln scans on a cron job

### PR DESCRIPTION
**Pull request by coilysiren**
## Summary

Solves 50% of #2496

## Changes proposed

- Sets up vuln scans on a cron job
- Posts it to slack

## Context for reviewers

I setup a highlight word on "CI Vulnerability Scans" so this bothers me specifically

## Testing

see `#internal-alerts-infra`

### CODE REVIEW

File: .github/workflows/ci-cron-vulnerability-scans.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# GitHub Actions CI workflow that runs vulnerability scans on the application's Docker image
+# to ensure images built are secure before they are deployed.
+
+name: CI Vulnerability Scans
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run every day at 07:00 UTC (3am ET, 12am PT) after engineers are likely done with work
+    - cron: "0 7 * * *"
```
Comment (chouinar) [inline .github/workflows/ci-cron-vulnerability-scans.yml:10]: What about running it closer to the start of the work day (7~9am EST?) just for all those fun last-second vulnerabilities that pop up.

Comment (coilysiren) [reply .github/workflows/ci-cron-vulnerability-scans.yml:10]: "fun" 😆 

anyways, yeah sure

**Comment by coilysiren** [comment]
I'm being blocked by actionlint :/ https://github.com/rhysd/actionlint/issues/471

File: .github/workflows/vulnerability-scans.yml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # GitHub Actions CI workflow that runs vulnerability scans on the application's Docker image
 # to ensure images built are secure before they are deployed.
 
+# RUN A TEST
+
```
Comment (chouinar) [inline .github/workflows/vulnerability-scans.yml:5]: Change looks good - just need to revert this

```suggestion
```

Comment (coilysiren) [reply .github/workflows/vulnerability-scans.yml:5]: oh thanks

**Review by mdragon — APPROVED** [review]

**Pull request merged**